woocommerce/woocommerce-android
# Login crash fix

Fixes #3618.

**To test:**
1. Create a site without Jetpack (using jurrasic.ninja, for example)
2. Start the login process by entering the site address
3. When the Jetpack-required screen is opened, rotate the screen
4. Notice the app doesn't crash

Update release notes:
